lang/cs/Org.Apache.REEF.Bridge/EvaluatorRequestorClr2Java.cpp

```diff
@@ -66,10 +66,31 @@ namespace Org {
 							  request->Number,
 							  request->MemoryMegaBytes,
 							  request->VirtualCore,
-							  JavaStringFromManagedString(env, request->Rack));
+							  JavaStringFromManagedString(env, request->Rack),
+							  JavaStringFromManagedString(env, request->RuntimeName));
 						  ManagedLog::LOGGER->LogStop("EvaluatorRequestorClr2Java::Submit");
 					  }
 
+					  array<byte>^ EvaluatorRequestorClr2Java::GetDefinedRuntimes() {
+						  ManagedLog::LOGGER->LogStart("EvaluatorRequestorClr2Java::GetDefinedRuntimes");
+						  JNIEnv *env = RetrieveEnv(_jvm);
+						  jclass jclassEvaluatorRequestor = env->GetObjectClass(_jobjectEvaluatorRequestor);
+						  jmethodID jmidGetDefinedRuntimes = env->GetMethodID(jclassEvaluatorRequestor, "getDefinedRuntimes", "()[B");
+
+						  if (jmidGetDefinedRuntimes == NULL) {
+							  fprintf(stdout, " jmidGetDefinedRuntimes is NULL\n");
```

Comment:
I followed the same convention as for other methods in this class. They all trace this part to stdout. I am not sure why but I prefer this method to behave in the same way as other methods in this class.